extensions/ffmpeg/src/main/jni/ffmpeg_jni.cc

```diff
@@ -201,13 +203,13 @@ AVCodec *getCodecByName(JNIEnv* env, jstring codecName) {
 }
 
 AVCodecContext *createContext(JNIEnv *env, AVCodec *codec,
-                              jbyteArray extraData) {
+                              jbyteArray extraData, jboolean use32BitFloatOutput) {
   AVCodecContext *context = avcodec_alloc_context3(codec);
   if (!context) {
     LOGE("Failed to allocate context.");
     return NULL;
   }
-  context->request_sample_fmt = OUTPUT_FORMAT;
+  context->request_sample_fmt = use32BitFloatOutput ? OUTPUT_FORMAT_32FLOAT : OUTPUT_FORMAT;
```

Comment:
Can this be read out of the `AVCodecContext` passed into `ffmpegDecode`, `decodePacket` and `ffmpegReset`, to avoid adding the extra parameter to those methods?